### ISSUE TITLE
what's the right way for theming this?

### ISSUE THREAD

**Issue by krtschmr** [fomantic/Fomantic-UI-SASS#2]
i'm coming from bootstrap, i wonder what would be the correct way to do in in semanticui?
i expected to just load the variables, then load custom variables/overrides and load the rest of the css to compile it. however:

there are like no variables
https://github.com/fomantic/Fomantic-UI-SASS/blob/master/app/assets/stylesheets/semantic-ui/globals/_variables.scss 

and colors etc are hardcoded
https://github.com/fomantic/Fomantic-UI-SASS/blob/master/app/assets/stylesheets/semantic-ui/elements/_loader.scss

the LESS file has it with variables: https://github.com/fomantic/Fomantic-UI/blob/master/src/definitions/elements/loader.less

if i read it correctly, this gem is a bit useless since it won't allow us customization as we should be able to do it?

**Comment by shanecav84** [comment]
You are correct that this gem doesn't support theming. I will take a look into adding it in.
